snuba/snuba_migrations/group_attributes/0003_add_first_release_id_to_group_attributes.py

```diff
@@ -0,0 +1,49 @@
+from typing import Sequence
+
+from snuba.clickhouse.columns import Column, UInt
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+from snuba.migrations.operations import OperationTarget, SqlOperation
+
+
+class Migration(migration.ClickhouseNodeMigration):
+    blocking = False
+
+    def forwards_ops(self) -> Sequence[SqlOperation]:
+        return [
+            operations.AddColumn(
+                storage_set=StorageSetKey.GROUP_ATTRIBUTES,
+                table_name="group_attributes_local",
+                column=Column(
+                    "group_first_release_id", UInt(8, Modifiers(nullable=True))
```

Comment:
Question, is this actually a full release ID? Is that really a UInt8 only? Aren't releases UUIDs?